### ISSUE TITLE
WARNING: /core/DataTable/Row.php(805) after update to 4.6.1

### ISSUE THREAD

**Issue by heurteph-ei** [piwik/piwik#18468]
## Expected Behavior
No warning

## Current Behavior
After the migration to version 4.6.1, some warnings are displayed in each page of the Matomo UI:
>WARNING: /core/DataTable/Row.php(805): Trying to add two strings in DataTable\Row::sumRowArray: 2021-12-07 01:50:26 + 2021-12-07 02:05:12 for column _metadata in row # [‘label’ => ‘Martijn’, ‘nb_visits’ => 73, ‘nb_actions’ => 82, ‘Actions_nb_pageviews’ => 79, ‘Goal_revenue’ => 0, ‘Goal_nb_conversions’ => 0, ‘_metadata’ => array ( ‘ts_archived’ => ‘2021-12-07 01:50:26’, ), ‘visits_evolution’ => 2.076, ‘visits_evolution_trend’ => 1, ‘actions_evolution’ => 1.504, ‘actions_evolution_trend’ => 1, ‘pageviews_evolution’ (Module: MultiSites, Action: getAllWithGroups, In CLI mode: false)

## Steps to Reproduce (for Bugs)
Migration from version 4.5.0 to version 4.6.1...
Then open Matomo Analytics...

## Context
From Forum:
https://forum.matomo.org/t/error-after-update-to-4-6-1/43990

## Your Environment

* Matomo new Version: 4.6.1
* Matomo previous Version: 4.5.0
* PHP Version: _To be completed_
* Server Operating System: _To be completed_
* Additionally installed plugins: _To be completed_

**Comment by heurteph-ei** [comment]
Some screen prints:
![](https://forum.matomo.org/uploads/db1181/optimized/2X/8/8e00328b141c68d18ed24912efb78c610527c800_2_690x432.png)

![](https://forum.matomo.org/uploads/db1181/optimized/2X/9/9975e548306ac0596b7719f88a1796c16a13c25d_2_690x388.jpeg)

**Comment by sgiehl** [comment]
@heurteph-ei Are you able to say which request fails on the dashboard? Should be visible in the browser console.

**Comment by tsteur** [comment]
FYI I would assume that the request itself maybe works but on a next page view these warnings show up? and the failed request is a different issue?

**Comment by heurteph-ei** [comment]
![](https://forum.matomo.org/uploads/db1181/original/2X/9/9f3e3626a96666db75ffa778dd54806ffbc089a4.png)
![](https://forum.matomo.org/uploads/db1181/original/2X/e/ee2ffbe49c8232a1b5efda2fc01d8bdc3ff4aa77.jpeg)
![](https://forum.matomo.org/uploads/db1181/original/2X/d/d6d492ce63e60a2ce439afccad8e32efa823100e.jpeg)
![](https://forum.matomo.org/uploads/db1181/original/2X/c/c71e502ca97a2d4a2afca217fe8314206ddba390.jpeg)

**Comment by heurteph-ei** [comment]
The user says also
>If I press reload browser the messages will be gone!, so next time I will copy the HTTP request error

**Comment by sgiehl** [comment]
@heurteph-ei This should be fixed with the next release. As we weren't able to reproduce that, feel free to reopen the issue or create a new one, if that still happens with the next release.

**Comment by twixi** [comment]
i have too error 4.6.2 and 
`WARNING: /core/DataTable/Row.php(805): Trying to add two strings in DataTable\Row::sumRowArray: 2021-12-18 12:41:58 + 2021-12-18 12:42:02 for column _metadata in row # ['label' =&gt; 'EG', 'nb_visits' =&gt; 65, 'nb_actions' =&gt; 133, 'Actions_nb_pageviews' =&gt; 110, 'Goal_revenue' =&gt; 0, 'Goal_nb_conversions' =&gt; 0, '_metadata' =&gt; array ( 'ts_archived' =&gt; '2021-12-18 12:41:58', ), 'visits_evolution' =&gt; -0.692, 'visits_evolution_trend' =&gt; -1, 'actions_evolution' =&gt; -0.835, 'actions_evolution_trend' =&gt; -1, 'pageviews_evolution' (Module: MultiSites, Action: getAllWithGroups, In CLI mode: false)`

`WARNING: /core/DataTable/Row.php(805): Trying to add two strings in DataTable\Row::sumRowArray: 0 + 2021-12-18 12:42:07 for column _metadata in row # ['label' =&gt; 'EG', 'nb_visits' =&gt; 70, 'nb_actions' =&gt; 142, 'Actions_nb_pageviews' =&gt; 119, 'Goal_revenue' =&gt; 0, 'Goal_nb_conversions' =&gt; 0, '_metadata' =&gt; array ( 'ts_archived' =&gt; 0, ), 'visits_evolution' =&gt; -1.04, 'visits_evolution_trend' =&gt; -2, 'actions_evolution' =&gt; -1.219, 'actions_evolution_trend' =&gt; -2, 'pageviews_evolution' =&gt; -1.255, 'pageviews_evolution_trend' (Module: MultiSites, Action: getAllWithGroups, In CLI mode: false)
`
and data not show...

![Зображення без підпису](https://user-images.githubusercontent.com/7196070/146641543-18bd1cff-5c44-4b4c-9d0a-8348ffc9ea18.png)

**Comment by Findus23** [comment]
@twixi Can you try applying the change from #18500 to see if it fixes it for you?

**Comment by twixi** [comment]
> I do not understand what needs to be done.... in #18500

**Comment by Sir-Will** [comment]
I'm still getting this error after the change from #18500.

```
WARNING: /core/DataTable/Row.php(806): Trying to add two strings in DataTable\Row::sumRowArray: 2022-01-10 00:06:28 + 2022-01-10 00:05:06 for column _metadata in row # ['label' =&gt; 'gaming', 'nb_visits' =&gt; 14459, 'nb_actions' =&gt; 63012, 'Actions_nb_pageviews' =&gt; 26368, 'Goal_revenue' =&gt; 0, 'Goal_nb_conversions' =&gt; 0, '_metadata' =&gt; array ( 'ts_archived' =&gt; '2022-01-10 00:06:28', ), 'visits_evolution' =&gt; 0.049, 'visits_evolution_trend' =&gt; 1, 'actions_evolution' =&gt; 0.02, 'actions_evolution_trend' =&gt; 1, 'pageviews_evol (Module: MultiSites, Action: getAllWithGroups, In CLI mode: false)

**Comment by tsteur** [comment]
@bx80 could there maybe be still an issue?

**Comment by bx80** [comment]
Thanks to everyone following up on this issue, it looks like the fix in #18500 didn't work. 

This is probably because unsummable columns are only checked for top level column names, if the column is an array (as it is in this case `_metadata`) then the array keys are not checked against the unsummable columns, so an attempt to sum up `ts_archived` is still being made even though it is meant to be ignored.

I will create another PR to also check array keys against the unsummable columns list when summing row data, which should hopefully resolve this warning message.
